go.mod

```diff
@@ -8,9 +8,9 @@ require (
 	github.com/hashicorp/go-getter v1.4.0
 	github.com/hashicorp/go-version v1.2.0
 	github.com/hashicorp/terraform-plugin-sdk v1.0.0
-	github.com/rancher/kontainer-driver-metadata v0.0.0-20191125162457-09a06565759f
-	github.com/rancher/rke v1.0.0
-	github.com/rancher/types v0.0.0-20191126013101-de3dc217d60f
+	github.com/rancher/kontainer-driver-metadata v0.0.0-20200116174051-6b21affb3bdd
+	github.com/rancher/rke v1.0.2
+	github.com/rancher/types v0.0.0-20191220141556-ad31d6815bbd
 	github.com/sirupsen/logrus v1.4.2
 	gopkg.in/ini.v1 v1.48.0 // indirect
 	gopkg.in/yaml.v2 v2.2.4
```

Comment:
If `go.mod` needs to be updated, could you please update vendor folder `GO111MODULE=on go mod vendor` and include it on the same commit with go.mod and go.sum??

Reply:
@rawmind0 I'm not that familiar with Go yet, apologies. I will do that as well.